src/lib/provable/field.ts

```diff
@@ -560,17 +569,14 @@ class Field {
    *
    * @example
    * ```ts
-   * Field(2).lessThan(3).assertEquals(Bool(true));
+   * let isTrue = Field(2).lessThan(3);
```

Comment:
some comments boyscouting in this file. I didn't like that the example for `lessThan()` was one that should actually have used `assertLessThan()`